src/app/inventory-page/StoreBuckets.tsx

```diff
@@ -74,11 +78,27 @@ export function StoreBuckets({
     ));
   }
 
+  const postMasterSpaceUsed = postmasterSpaceUsed(stores[0]);
+  const checkPostmaster = bucket.hash === BucketHashes.LostItems;
+  if (!checkPostmaster) {
+    // Only the postmaster needs a header per store, the rest span across all stores
+    stores = [stores[0]];
+  }
+
```

Comment:
Ah right, mobile makes sense.

```suggestion
  const postMasterSpaceUsed = postmasterSpaceUsed(stores[0]);
  const checkPostmaster = bucket.hash === BucketHashes.LostItems;
```
This assignment is now unused.